apps/condo/pages/ticket/index.tsx

```diff
@@ -40,6 +40,7 @@ import { TablePageContent } from '@condo/domains/common/components/containers/Ba
 import { useWarrantySearch } from '@condo/domains/ticket/hooks/useWarrantySearch'
 import { useFiltersTooltipData } from '@condo/domains/ticket/hooks/useFiltersTooltipData'
 import { useReturnedSearch } from '@condo/domains/ticket/hooks/useReturnedSearch'
+import _ from 'lodash'
```

Comment:
it's better to use `isString`. or `lodash/isString` or `typeof === 'string'` to reduce the bundle size